Environment-Definitions/ARMTemplates/App_Service_with_Cosmos/environment.yaml

```
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://github.com/Azure/deployment-environments/releases/download/2022-11-11-preview/manifest.schema.json
+name: App-Service-with-Cosmos
+version: 1.0.0
+summary: Azure APP Service with Cosmos Environment
+description: Deploys an Azure App Service with Cosmos, it is compatible with azd.
+runner: ARM
+templatePath: azuredeploy.json
+
+parameters:
+  - id: runtimeType
+    name: runtimeType
```

Please provide a more user friendly display name like `Runtime Type`
```suggestion
    name: Runtime Type
```